lib/resources/r4/related_person.yaml

```diff
@@ -194,7 +195,7 @@ fields:
           <li> Up to two `given` names may be provided. If additional given names are required they must be provided in the second value as a single string separated by blank spaces. </li>
           <li> Only one `prefix` and one `suffix` may be provided. </li>
           <li> `period.end` is not supported. </li>
-          <li> When specifying a `period`, `period.start` must include a time component with a timezone. </li>
+          <li> When specifying a period, the fields must include a time component with a timezone. </li>
```

Comment:
Just a NOTE:  The note saying that the time component is with a timezone makes sense, as it seems like 000Z represents the Zulu time zone.UTC +0

Reply:
Good to know. Thanks for checking on that.

Reply:
Added a note in a relevant comment here about this update - https://github.com/cerner/fhir.cerner.com/pull/650#discussion_r643189624